src/main/java/edu/cornell/kfs/ksr/service/impl/SecurityRequestDocumentServiceImpl.java

```diff
@@ -138,6 +146,20 @@ protected void buildSecurityRequestRoles(SecurityRequestDocument document) {
         document.setSecurityRequestRoles(requestRoles);
     }
     
+    protected boolean isKSRAllowedToEditRoleQualifersForRole(String roleId) {

```

Comment:
Now that you changes the parameter name, should the method name also be updated? Thank you

Reply:
OK